.eslintrc.json

```diff
@@ -7,21 +7,40 @@
   },
   "extends": [
     "plugin:react/recommended",
-    "standard"
+    "standard",
+    "plugin:import/typescript"
   ],
   "ignorePatterns": "scripts",
-  "overrides": [{
-    "files": ["*.ts", "*.tsx"],
-    "rules": {
-      "semi": "off",
-      "Extra semicolon": "off",
-      "import/prefer-default-export": "off",
-      // temporary fix from 'typescript-elint' docs
-      // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unused-vars.md
-      "no-unused-vars": "off",
-      "@typescript-eslint/no-unused-vars": ["error"]
+  "overrides": [
+    {
+      "files": ["**/*.ts", "**/*.tsx"],
+      "env": { "browser": true, "es6": true, "node": true },
+      "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended"
+      ],
+      "globals": { "Atomics": "readonly", "SharedArrayBuffer": "readonly" },
+      "parser": "@typescript-eslint/parser",
+      "parserOptions": {
+        "ecmaFeatures": { "jsx": true },
+        "ecmaVersion": 2018,
+        "sourceType": "module",
+        "project": "./tsconfig.eslint.json"
```

Comment:
Any reason to have a separate `tsconfig.json` and `tsconfig.eslint.json`?

Reply:
Merged them in the latest commit